easybuild/easyconfigs/n/NiBabel/NiBabel-2.3.0-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'NiBabel'
+version = '2.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://nipy.github.io/nibabel'
+description = """NiBabel provides read/write access to some common medical and neuroimaging file formats,
+ including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT
+ as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and
+ annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Pillow', '5.3.0', versionsuffix),
+]
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_defaultclass = 'PythonPackage'
+exts_list = [
+    ('pydicom', '1.2.0', {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/pydicom/pydicom/archive/'],
+        'checksums': ['56068467f6d339223349717fe2a4f6edd219bb299eed8f15a9a875a357942949'],
+    }),
+    ('nibabel', version, {
+        'source_urls': ['https://pypi.python.org/packages/source/n/nibabel'],
+        'checksums': ['bf34aeb0f7ca52dc528ae4f842607cea307b334163857ff1d64d43068f637ada'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/nib-dicomfs', 'bin/nib-ls', 'bin/nib-nifti-dx', 'bin/parrec2nii'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/nibabel', 'lib/python%(pyshortver)s/site-packages/nisext'],
+}
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```

Comment:
The modextrapaths for PYTHONPATH is not needed with a PythonBundle and can be removed.

Reply:
removed the PYTHONPATH setting.